### PR TITLE
PKU-174-175

### PR DESCRIPTION
**Pull request by dephillipsmichael**
Add task checks to end of study screen

### CODE REVIEW

File: BiomarinPKU/BiomarinPKU/MotorControl+Bridge.swift [hunk 1]
```diff
@@ -48,6 +48,22 @@ public extension RSDIdentifier {
...
+        return self == .taskSwitchTask ||
+                self == .attentionalBlinkTask ||
+                self == .symbolSubstitutionTask ||
+                self == .spatialMemoryTask ||
+                self == .nBackTask ||
+                self == .goNoGoTask
+    }
+    
+    func isActiveTask() -> Bool {
+        return self == .tappingTask ||
```
Comment (DanWebster) [inline BiomarinPKU/BiomarinPKU/MotorControl+Bridge.swift]: @dephillipsmichael don't need to have the tapping task in here for the permissions check, b/c it is option for proceeding.  Don't want BMRN to test this for its optionality and then open up a new issue.  Is this function used anywhere else besides in the permissions check on the end of study or could we remove the tap test from this check?

Comment (dephillipsmichael) [reply BiomarinPKU/BiomarinPKU/MotorControl+Bridge.swift]: It is not used anywhere else, but I will push that functionality to the EndOfStudyViewController so it isn't confusing.

Comment (dephillipsmichael) [reply BiomarinPKU/BiomarinPKU/MotorControl+Bridge.swift]: Done

**Comment by DanWebster** [comment]
Works great, End of Study screens responsive to motion sensing (for dual phone hold but not tapping) and connectivity (for all cognitive tests).

**Pull request merged**